cortex_certifai_azure_notebooks_NY_Taxi/Tutorial.ipynb

```diff
@@ -72,7 +72,7 @@
   },
```

Comment:
in `Usage/Preparation Steps`
1. Running on Azure Notebook portal
   3. `Upload the cat_encoder.py from certifai_toolkit/examples/notebooks` now is not needed since, cat _encoder is now a part of the repo.   can be deleted

Reply:
also `4. Upload the scanner, engine and from certifai_toolkit/packages folder` doesn't mention to install `common` which is also needed for certifai imports

Reply:
Will remove and add (respectively!)